index.html

```diff
@@ -1,338 +1,257 @@
+        <a target = none href = "https://www.youtube.com/watch?v=Oz9zw7-_vhM">https://www.youtube.com/watch?v=Oz9zw7-_vhM</a> <br />
+        <a target = none href = "https://www.alpha-sense.com/blog/faang-q3-earnings/">https://www.alpha-sense.com/blog/faang-q3-earnings/</a> <br />
+        <a target = none href = "https://www.newsweek.com/fact-check-elon-musk-buying-twitter-led-white-house-target-section-230-1701449">https://www.newsweek.com/fact-check-elon-musk-buying-twitter-led-white-house-target-section-230-1701449</a> <br />
+        <a target = none href = "https://www.creativereview.co.uk/why-nfts-are-the-future-of-creative-expression/">https://www.creativereview.co.uk/why-nfts-are-the-future-of-creative-expression/</a><br/>
+        <a target = none href = "https://www.wsj.com/articles/how-elon-musk-can-liberate-twitter-censorship-fact-check-free-speech-buy-website-platform-takeover-content-moderation-hate-speech-misinformation-disinformation-11650983718">https://www.wsj.com/articles/how-elon-musk-can-liberate-twitter-censorship-fact-check-free-speech-buy-website-platform-takeover-content-moderation-hate-speech-misinformation-disinformation-11650983718</a><br/>
+        <a target = none href = "https://www.brookings.edu/blog/up-front/2020/07/06/how-misinformation-spreads-on-twitter/">https://www.brookings.edu/blog/up-front/2020/07/06/how-misinformation-spreads-on-twitter/</a><br/>
+        <a target = none href = "https://www.youtube.com/watch?v=jEK7P4-rDk4">https://www.youtube.com/watch?v=jEK7P4-rDk4</a><br/> 
+        <a target = none href = "https://www.americanbar.org/groups/crsj/publications/human_rights_magazine_home/the-ongoing-challenge-to-define-free-speech/in-the-age-of-socia-media-first-amendment/">https://www.americanbar.org/groups/crsj/publications/human_rights_magazine_home/the-ongoing-challenge-to-define-free-speech/in-the-age-of-socia-media-first-amendment/</a><br />
+    </ul>
+</article>
+</body>
+
+<style>
+body {
+    background-color: rgb(168, 235, 255);
+    font-weight: bold;
+    color: rgb(255, 0, 0);
+}
+
+article {
+    width: 80%;
+    margin: 0 auto;
+}
+
+h1 {
+    font-family: garamond;
+    font-size: xx-large;
+    padding-bottom: 10px;
+    border-bottom: 3px solid rgb(255, 255, 255);
+    margin-top: 40px;
+}
+
+h2{
+    font-family: 'Courier New', Courier, monospace;
+    font-size: xx-large;
+    padding-bottom: 10px;
+    margin-top: 40px;
+}
+
+.Protestor{
+    text-align: center;
+}
+
+p{
+    line-height: 24px;
+    text-align: center;
+    font-family: 'Courier New', Courier, monospace;
+}
+
+ul{
+    line-height: 24px;
+    text-align: left;
+    font-family: 'Courier New', Courier, monospace;
+}
+
+.listToggle:hover + .hiddenList{
+    display: block;
+}
+.hiddenList:hover{
+    display: block;
+}
+
+.hiddenList{
+    display: none;
+}
+
+#sources{
+    display: none;
+}
+
+#sourceInfo{
+    color: purple;
+}

```

Comment:
love all the interactive components here, hiding the sources initially helps keep the page tidy && the playful sentence swapping on click at the top is a real interesting experiment (excited to discuss details like this w/the class on monday), however, i only knew to click on them b/c i looked at the code && noticed u had `onclick` event handler attributes on those elements, otherwise i wouldn't have thought to do so.

i suggest adding some CSS that might prompt the user to click, for example by creating "hover" states for some of these elements which change their CSS on hover, for example:
```css
#sources:hover {
  color: red;
  cursor: pointer;
}

#udhr1:hover {
  color: green;
  cursor: pointer;
}
```
obviously it doesn't need to be those exact properties, but some sort of visual shift on hover would be helpful